docs/en/04_Changelogs/4.0.0.md

```diff
@@ -1178,20 +1178,20 @@ class MyAddressField extends
 }
 ```
 
-### `DataObject::database_fields` or `DataObject::db` {#dataobject-db-database-fields}
+### Removed `DataObject::database_fields` or `DataObject::db` {#dataobject-db-database-fields}
 
-These methods have been updated to include base fields (such as ID, ClassName, Created, and LastEdited), as
-well as composite DB fields.
+The methods `DataObject::database_fields()`, `DataObject::custom_database_fields()` and `DataObject::db()` have
+been removed.
+
+Instead, to get all database fields for a dataobject, including base fields (such as ID, ClassName, Created, and LastEdited), use `DataObject::getSchema()->databaseFields($className, $aggregate = true)`.
+To omit the base fields, pass a value of `false`  as the `$aggregate` parameter, e.g. `DataObject::getSchema()->databaseFields(Member::class, false)`.
 
 `DataObject::database_fields` does not have a second parameter anymore, and can be called directly on an object
```

Comment:
Need to change this reference as well.

Reply:
Fixed. Not sure how that found its way back in!